packages/experimental-utils/src/ts-eslint/Rule.ts

```diff
@@ -179,7 +179,7 @@ interface RuleContext<
    * The shared settings from configuration.
    * We do not have any shared settings in this plugin.
    */
-  settings: {};
+  settings: Record<string, any>;
```

Comment:
```suggestion
  settings: Record<string, unknown>;
```

Reply:
I think it's great!

For some reason I had a memory of seeing a comment mentioning that unknown can be used once TS2 is dropped, but you use `unknown` in other places so I must have been getting confused with DT or something 🤷‍♂